Forwarded: not-needed
 Breaks functions (visible adhoc ssid) and thus cannot be forwarded.

--- a/net/mac80211/ibss.c
+++ b/net/mac80211/ibss.c
@@ -83,10 +83,12 @@ ieee80211_ibss_build_presp(struct ieee80
 
 	pos = (u8 *)mgmt + offsetof(struct ieee80211_mgmt, u.beacon.variable);
 
+	/*
 	*pos++ = WLAN_EID_SSID;
 	*pos++ = ifibss->ssid_len;
 	memcpy(pos, ifibss->ssid, ifibss->ssid_len);
 	pos += ifibss->ssid_len;
+	*/
 
 	sband = local->hw.wiphy->bands[chandef->chan->band];
 	rate_flags = ieee80211_chandef_rate_flags(chandef);
